editors/code/src/toolchain.ts

```diff
@@ -158,9 +157,9 @@ export const getPathForExecutable = memoize(
         try {
             // hmm, `os.homedir()` seems to be infallible
             // it is not mentioned in docs and cannot be infered by the type signature...
-            const standardPath = path.join(os.homedir(), ".cargo", "bin", executableName);
+            const standardPath = vscode.Uri.joinPath(vscode.Uri.file(os.homedir()), ".cargo", "bin", executableName);
 
-            if (isFile(standardPath)) return standardPath;
+            if (isFile(standardPath.path)) return standardPath.path;
```

Comment:
We convert an `Uri` to a string here, then convert it back to an `Uri`. And I'm not sure what happens if the path doesn't exist -- is there a way to `require("vscode")` in the Developer Tools?

Reply:
> is there a way to require("vscode") in the Developer Tools?

I would expect not. The developer tools run inside the chromium based part of electron. The extensions run in the node.js based part.

Reply:
@wxb1ank I haven't looked at the code outside of the diff, but do you think this can be simplified?